README.md

```diff
@@ -366,6 +366,10 @@ const connector = new Connector({
 });
```

Comment:
I still wonder if we should update the basic usage to show adding the sqladmin scopes here?
```js
const connector = new Connector({
  auth: new GoogleAuth({
    scopes: ['https://www.googleapis.com/auth/sqlservice.admin']
  }),
});
```

Reply:
Yes, definitely.